portfolio/src/main/webapp/statistics.html

```diff
@@ -24,7 +24,7 @@
     <link href="css/styles.css" rel="stylesheet" />
     <link href="css/custom.css" rel="stylesheet" />
   </head>
-  <body id="page-top" onload="statisticsOnLoad()">
+  <body id="page-top" onload="onLoad()">
```

Comment:
not too sure if I like this. Could you just add the searchHash function to statistics onLoad? because every other page uses onLoad and I don't want to unnecessarily load mapsor charts in nonexistent DOM elements in other pages.

Reply:
Yeah for sure.